app/assets/v2/scss/gitcoin.scss

```diff
@@ -519,8 +525,13 @@
     }
   }
 
-  .accordion-panel  {
+  .accordion-panel {
     padding-left: 28px;
+
+    p {
+      margin-bottom: 0;
```

Comment:
can't we use mb-0 ?

Reply:
this css is applied globally on all accordions...hence `gitcoin.scss`